ios/Mappers.swift

```diff
@@ -833,6 +833,14 @@ class Mappers {
 
         return (["collectInputResults": collectInputResults])
     }
+
+    class func mapFromReaderEvent(_ readerEvent: ReaderEvent) -> String {
+        switch readerEvent {
+            case ReaderEvent.cardInserted: return "cardInserted"
+            case ReaderEvent.cardRemoved: return "cardRemoved"
+            default: return "unknown"
+        }
```

Comment:
recommend using `@unknown default` so the compiler tells us when new entries are added

and should be able to remove the prefix

```suggestion
        switch readerEvent {
            case .cardInserted: return "cardInserted"
            case .cardRemoved: return "cardRemoved"
            @unknown default: return "unknown"
        }
```

Reply:
and should the values be the values from the enum?

```
export enum ReaderEvent {
  CARD_INSERTED = 'CARD_INSERTED',
  CARD_REMOVED = 'CARD_REMOVED',
}
```

? (not sure what's expected on the RN layer there - or does something else map them to that?)